.env.example

```diff
@@ -1,3 +0,0 @@
-TWILIO_ACCOUNT_SID=ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Is this file deleted? That might be most straightforward.

Reply:
Yes the file is deleted

Reply:
You can verify this by [viewing the branch in file view](/Twilio-org/phonebank/tree/remove-dotenv). Github doesn't make this very clear in the diff view, do they?